apps/src/javalab/JavalabEditor.jsx

```diff
@@ -61,16 +64,38 @@ class JavalabEditor extends React.Component {
   }
 
   componentDidMount() {
+    const {isDarkMode} = this.props;
+    const extensions = [...editorSetup];
+
+    if (isDarkMode) {
+      extensions.push(tagExtension('style', oneDark));
+    } else {
+      extensions.push(tagExtension('style', lightMode));
+    }
     this.editor = new EditorView({
       state: EditorState.create({
         doc: this.props.editorText,
-        extensions: editorSetup
+        extensions: extensions
       }),
       parent: this._codeMirror,
       dispatch: this.dispatchEditorChange()
     });
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.isDarkMode !== this.props.isDarkMode) {
+      if (this.props.isDarkMode) {
+        this.editor.dispatch({
```

Comment:
Should this be using `dispatchEditorChange`?

Reply:
I think that's for real editor changes while this is for extension changes according to the Configuration section on [this page](https://codemirror.net/6/docs/migration/)

Reply:
👍